communication/slack-backup/README.md

```diff
@@ -0,0 +1,136 @@
+
+![screenshot of looking at the cookie in Chrome web developer tools](getcookie.jpeg)
+
+Enter "kubernetes" for the workspace name, your session cookie, and select "Yes" for "Confirm creation of workspace" to create credentials.
+
+![screenshot of the create credentials from cookie interface](createcredential.jpeg)
+
+Now you can export your private Channels and DMs. The easiest way is to use ChannelIDs.
+
+To retrieve a ChannelID of a private channel, select the channel and click on the 3 dots to choose "Open channel details". The popup will display the Channel ID at the bottom.
+
+![showing "open channel details" on a channel](channelid1.jpeg)
+
+![showing the channelid at the bottom of the dialog](channelid2.jpeg)
+
+For DMs, you follow a similar method. Click the three dots and select "Open conversation details". Afterwards you can copy a ChannelID.
+
+![selecting channel details from a direct message](channelid3.jpeg)
+
+![showing where the channel id is in the dialog](channelid4.jpeg)
+
+Now you can export the channel contents, selecting each channel via its ID. We recommend the "standard" storage type.
+
+![screenshot of the UI showing the options for exporting a channel](exportdata1.jpeg)
+
+Once you have selected all the options, hit ESC to go back, and choose "Start Export". Slackdump will do a lot of pulling data, and eventually create a zip file in the root of your home directory.
+
+![example of Slackdump exporting a channel](exportdata2.jpeg)
+
+## View the export:
+
+Once the workspace data is dumped, you can run built-in viewer:
+
+```shell
+slackdump view <zip or directory>
+```
+
+The built-in viewer supports all types of dumps:
+
+1. Slackdump Archive format
+1. Standard and Mattermost Slack Export
+1. Dump mode files
+
+Alternatively, you can use one of the following tools to view the
+export results:
+
+- [SlackLogViewer] - a fast and powerful Slack Export viewer written in C++, works on Export files (images won't be displayed, unless you used an export token flag).
+- [Slackdump2Html] - a great Python application that converts Slack Dump to a
+  static browsable HTML. It works on Dump mode files.
+- [slack export viewer][slack-export-viewer] - Slack Export Viewer is a well known viewer for
+  slack export files. Supports displaying files if saved in the "Standard" file mode.
+
+[SlackLogViewer]: https://github.com/thayakawa-gh/SlackLogViewer/releases
+[Slackdump2Html]: https://github.com/kununu/slackdump2html
+[slack-export-viewer]: https://github.com/hfaran/slack-export-viewer
+
+## What Should I Do with the Backup Files?
+
+**Kubernetes Private Channels**: For most backups, contact #sig-contribex about saving the file(s) to the Kubernetes-owned Google Drive. Let them know who should have access to the files. If the private channel is such that the ContribEx leads should not see the contents of it, discuss options with them.
```

Comment:
I think a better approach is to ask us (me wearing the ContribEx hat) to get a folder in the shared drive for the group in which the DM belongs to. I am wary of us (again wearing the ContribEx hat) handling sensitive communications that we otherwise won't have access to.